homework.4/homework.4.playground/Contents.swift

```diff
@@ -0,0 +1,55 @@
+// homework.4
+
+import UIKit
+
+var tupleArray = [(100, "Continue"), (202, "Accepted"), (203, "Non-Authoritative Information"),
+                  (302, "Found"), (307, "Temporary Redirect"), (404, "Not Found")]
+
+// Task 1
+
+func squaringIntegerElement(_ : [(Int, String)]) -> [Int] {
```

Comment:
There's no need for `ing` ending in the functions naming. So it's better to have `squareIntegerElement` here.

You ignore the parameter name here. You can't use this parameter inside this function. So this code is absolute. You need to give this parameter a name and use it inside the function. Right now you just use a parameter from the outside